### PR TITLE
FIPS_ENABLED-management cluster

### PR DESCRIPTION
**Pull request by huangmingxia**
Add hypershift fips compliance CI test job - management cluster
User story: [HOSTEDCP-163](https://issues.redhat.com/browse/HOSTEDCP-163)
test case: [OCP-43554](https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-43554)
QE process: [OCPQE-14059](https://issues.redhat.com/browse/OCPQE-14059)

### CODE REVIEW

**Comment by huangmingxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-ovn-hypershift-fips-mgmt-p1-f14

**Comment by huangmingxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-ovn-hypershift-fips-mgmt-p1-f14

**Comment by huangmingxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-ovn-hypershift-fips-mgmt-p1-f14

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 1]
```diff
@@ -680,6 +680,19 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-efs
+- as: aws-ipi-ovn-hypershift-fips-mgmt-p1-f14
+  cron: 52 14 14,29 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      FIPS_ENABLED: "true"
```
Comment (LiangquanLi930) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:689]: Don't we need to add `EXTRA_ARGS: --fips`?

**Pull request closed without merging**